mteb/get_tasks.py

```diff
@@ -103,8 +105,8 @@ def get_tasks(
 
     Examples:
         >>> get_tasks(languages=["eng", "deu"], script=["Latn"], domains=["Legal"])
-        >>> get_tasks(languages=["eng"], script=["Latn"], task_type="Classification")
-        >>> get_tasks(languages=["eng"], script=["Latn"], task_type="Clustering", exclude_superseeded_datasets=False)
+        >>> get_tasks(languages=["eng"], script=["Latn"], task_types=["Classification"])
+        >>> get_tasks(languages=["eng"], script=["Latn"], task_types=["Clustering"], exclude_superseeded_datasets=False)
```

Comment:
I think that making task_types a list will break some scripts that use this function, see: _docs/create_tasks_table.py_.
Have you tried running the tests locally with `make test` or `make pr`?

Reply:
Not sure if I follow. `create_tasks_table` doesn't contain any call to `get_tasks` with `task_types` argument. If anything I made this example consistent with typing above. Or in fact I made it work. Previosuly:
```
> get_tasks(languages=["eng"], script=["Latn"], task_types="Clustering", exclude_superseeded=False)
[]
```

now:
```
> get_tasks(languages=["eng"], script=["Latn"], task_types=["Clustering"], exclude_superseeded=False)
[ArxivClusteringP2P(name='ArxivClusteringP2P', languages=['eng']),
....
 MasakhaNEWSClusteringS2S(name='MasakhaNEWSClusteringS2S', languages=['amh', 'eng', 'fra', '...'])]
```
PS I caught another bug here `exclude_superseeded_datasets` -> `exclude_superseeded`

>  Have you tried running the tests locally with make test or make pr?

Yes, but OOC do my local test differ at all form test run by CI? all were green below.

Reply:
Okay, I thougth _doc/create_tasks_table.py_ uses filters, but for now the filters are not used so this change won't break anything